File: Período_1/Lista 4/C/L4_Q13.c
#include <stdio.h>

int main(void){

    int valor_empre, parcelas, valor_parcelas, salario;
 
    printf("Valor do Emprestimo: ");
    scanf("%d", &valor_empre);

    printf("Numero de Parcelas: ");
    scanf("%d", &parcelas);

    printf("Salario: ");
    scanf("%d", &salario);

    
    valor_parcelas = valor_empre / parcelas;

    if(valor_parcelas <= salario * 0.3)
        printf("APROVADO");
    else
        printf("NAO APROVADO");

    return 0;
}

File: Período_1/Lista 4/Python/L4_Q13.py
valorEmprestimo = float(input("Informe o valor do empréstimo: "))
nParcelas = int(input("Informe quantidade de parcelas: "))
salario = float(input("Informe o seu salario: "))

if valorEmprestimo / nParcelas <= salario * 0.30:
    print(f"Imprestimo foi aprovado!!")
else:
    print(f"Imprestimo não foi aprovado T.T")